pythonwhois/parse.py

```diff
@@ -770,6 +765,21 @@ def parse_dates(dates):
 							# Unknown month name, default to 0
 							month = 0
 
+					# .fr has not year (registered for one year)
+					try:
+						year = int(result.group("year"))
+					except IndexError:
+						now = datetime.date.today()
+						year = now.year
+						if now.month < month:
```

Comment:
Wouldn't this cause issues if the current month is the month of expiry - ie. the domain expires in a few days?

I think it's better to just compare dates - in Python, this should be possible with the `<` and `>` operators like for any other value. Something like checking whether the day/month has already occurred in the current year or not.